internal/deb/extract_test.go

```diff
@@ -60,8 +66,56 @@ var extractTests = []extractTest{{
 		"/etc/":               "dir 0755",
 		"/etc/os-release":     "symlink ../usr/lib/os-release",
 	},
+	createdPaths: []string{
```

Comment:
Thanks for adding these tests. Can we please just invert the logic and call this as "notCreated" instead, so that what is highlighted here is the exception, what is missing?   Otherwise it looks like we're just saying the exact same thing repeatedly, which would be pointless, and it becomes an exercise for the reader to do the delta in their heads and picture the rationale.

Reply:
Changed! Hopefully the tests are more clear now.